dali/image/jpeg.cc

```diff
@@ -118,10 +131,6 @@ DALIError_t DecodeJPEGHost(const uint8 *jpeg, int size,
   } else {
     DALI_RETURN_ERROR("Unsupported image type.");
   }
-  int h, w;
-  int c = (type == DALI_GRAY) ? 1 : 3;
-
-  DALI_CALL(GetJPEGImageDims(jpeg, size, &h, &w));
 
 #ifndef NDEBUG
```

Comment:
Please move this debug code to line 116 - between GetJPEGImageDims and image->Resize